src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -491,7 +492,14 @@ function updateConstSumMessageQn(qnNum) {
     const numRecipients = parseInt($(`[name="questionresponsetotal-${qnNum}"]`).val(), 10);
     const distributeToRecipients = $(`#constSumToRecipients-${qnNum}`).val() === 'true';
     const pointsPerOption = $(`#constSumPointsPerOption-${qnNum}`).val() === 'true';
-    const forceUnevenDistribution = $(`#constSumUnevenDistribution-${qnNum}`).val() === 'true';
+
+    const constSumDistributePointsOptionSelected = $(`#constSumDistributePointsOptions-${qnNum}`).val();
+    const forceUnevenDistribution = constSumDistributePointsOptionSelected
```

Comment:
personally i find this variable confusing. Since the only place it is used is at `updateConstSumMessageQn` and the way it is used is `!forceUnevenDistribution`, which leads to a double-negation (one in the  definition one in the usage), I think you can change it into `noForceUnevenDistribution = constSumDistributePointsOptionSelected === ParamsNames.FEEDBACK_QUESTION_CONSTSUMNOUNEVENDISTRIBUTION;` and at its usage change it into `forceUnevenDistribution`.